tests/components/upnp/test_init.py

```diff
@@ -49,6 +50,9 @@ async def test_async_setup_entry_default(hass: HomeAssistant) -> None:
     entry.add_to_hass(hass)
     assert await hass.config_entries.async_setup(entry.entry_id) is True
 
+    coordinator: UpnpDataUpdateCoordinator = entry.runtime_data
+    coordinator.device._igd_device.async_subscribe_services.assert_called()
```

Comment:
We should avoid accessing `runtime_data` in the tests, instead patch `async_subscribe_services` or if its already mocked somewhere else, assert using that mock.